script/vsts/platforms/windows.yml

```diff
@@ -99,6 +99,23 @@ jobs:
       testRunTitle: Windows $(buildArch)
     condition: ne(variables['Atom.SkipTests'], 'true')
 
+  - script: |
+      IF NOT EXIST "%ARTIFACT_STAGING_DIR%\crash-reports" MKDIR "%ARTIFACT_STAGING_DIR%\crash-reports"
+      IF EXIST "%Temp%\Atom Crashes" (
+        FOR %%a in ("%Temp%\Atom Crashes\*.dmp") DO XCOPY "%%a" "%ARTIFACT_STAGING_DIR%\crash-reports" /I
+      )
```

Comment:
For future reference, you're not tied to batch here; you can use [`powershell`](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops) to write in PowerShell instead:

```yaml
- powershell: |
    if (Path-Exists "${env:ARTIFACT_STAGING_DIR}\crash-reports") {
      Create-Item -Type Directory "${env:ARTIFACT_STAGING_DIR}\crash-reports"
    }
    # etc
```